locale/en/blog/community/v5-to-v7.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Farewell to Node.js v5, Preparing for v7
+date: 2016-08-30T06:57:53.463Z
+status: publish
+category: Annoucements
+slug: v5-to-v7
+layout: blog-post.hbs
+author: Rod Vagg
+---
+
+You may have missed it but at the end of June, the Node.js project said a final farewell to version 5. There will be no more patches, critical or otherwise, for this branch. To those who have been using Node.js for some time this may seem anomalous, shouldn't major versions stick around for _years_?
+
+## We have a plan!
+
+![LTS Schedule Summary](/static/images/blog/201609_lts_schedule_summary.gif)
+
+Last year, the core team devised an LTS and release plan to balance the various wants and needs expressed by Node.js users. Chief among those were:
```

Comment:
`<abbr title="Long Term Support">LTS</abbr>` for this first instance of the acronym would be nice.